apps/laboratory/src/pages/library/ethers.tsx

```diff
@@ -17,9 +17,10 @@ const modal = createAppKit({
   defaultNetwork: mainnet,
   projectId: ConstantsUtil.ProjectId,
   features: {
-    analytics: true
-  },
-  customWallets: ConstantsUtil.CustomWallets
```

Comment:
is this ok?

Reply:
+ we'd better to left the `socials` and `email` values default